src/shared/Entities/reducer.js

```diff
@@ -21,7 +21,15 @@ function mergeEntities(entities, newEntities) {
 // deletes all items from entities with matching key, id in deleteEntities
 function deleteEntities(entities, deleteEntities) {
   return mapValues(entities, function (value, key) {
-    // eslint-disable-next-line
+    //RA Summary: eslint - security/detect-object-injection - Using square bracket notation with user input can lead to exploitation
+    //RA: Uses object square bracket notation
```

Comment:
Trying to keep with convention...

```suggestion
    //RA Summary: eslint - security/detect-object-injection
    //RA: Using square bracket notation with user input can lead to exploitation
    //RA: Uses object square bracket notation
```

Reply:
Ah sorry! Noted for the future